modules/understanding-agent-install.adoc

```diff
@@ -61,10 +61,13 @@ Recommended cluster resources for the following topologies:
 |HA cluster|3 to 5|2 and above |8 vCPUs|16 GB of RAM|120 GB
 |====
 
+// These supported platforms are also documented in nodes/nodes/nodes-nodes-adding-node-iso.adoc
```

Comment:
Do we need to add installation-configuration-parameters.adoc
```suggestion
// These supported platforms are also documented in nodes/nodes/nodes-nodes-adding-node-iso.adoc and installation-configuration-parameters.adoc
```

Reply:
Sure, I don't see how it would hurt to be extra informative here.